src/lib/containers/widgets/themes-plot/ThemesPlot.tsx

```diff
@@ -287,97 +293,100 @@ const ThemesPlot: FunctionComponent<ThemesPlotProps> = ({
               </div>
             </div>
           ))}
-
-          <div className="ThemesPlot__dotPlot">
-            {yLabelsForDotPlot.map((label, i) => (
-              <div
-                key={`plotDiv_${+i}`}
-                className="ThemesPlot__dotPlot__row"
-                style={{
-                  backgroundColor: dotPlot.plotStyle?.backgroundColor,
-                }}
-              >
-                <div className="ThemesPlot__dotPlot__barColumn">
-                  <ElementWithTooltip
-                    idForToolTip={`plotDiv1_${+i}`}
-                    tooltipText={`${getTooltip(dotPlotQueryData, label)} `}
-                    tooltipVisualProps={tooltipProps}
-                    callbackFn={() => _.noop}
-                  >
-                    <div>
-                      <span className="ThemesPlot__dotPlot__themeLabel">
-                        {label}
-                      </span>
-                      <br />
-                      <span className="ThemesPlot__dotPlot__countLabel">
-                        {totalsByDotPlotY[i].count} {sideBarPlot.countLabel}
-                      </span>
-                      <br />
-                      <BarPlot
-                        style={{ width: '100%' }}
-                        layoutConfig={barLayoutConfig}
-                        optionsConfig={optionsConfig}
-                        plotData={sideBarPlotData}
-                        isTop={false}
-                        xMax={xMaxForSideBarPlot}
+          <div style={{ display: 'flex', position: 'relative'}}>
+            <div className="ThemesPlot__dotPlotYLabel">{dotPlotYAxisLabel}</div>
```

Comment:
This is the only major change.. the resulting diff is mostly reshuffling of HTML since the extra div was added

Reply:
I'd argue that your __dotPlotYLabel class is doing most of the work!